test/ui/ResultListTest.ts

```diff
@@ -143,6 +143,25 @@ export function ResultListTest() {
       });
     });
 
+    it('should tell if there are more results to display after a successful query', done => {
+      Simulate.query(test.env);
+      Defer.defer(() => {
+        expect(test.cmp.hasPotentiallyMoreResultsToDisplay()).toBeTruthy();
+        done();
+      });
+    });
+
+    it('should tell if there are no more results to display after a successful query with a limited amount of results returned', done => {
+      const results = FakeResults.createFakeResults(5);
```

Comment:
Similarly here, instead of arbitrarily using `5`, we could use `settings.numOfResultsBeforeScroll`.